pkg/kubelet/pluginmanager/pluginwatcher/plugin_watcher.go

```diff
@@ -77,13 +72,6 @@ func (w *Watcher) Start(stopCh <-chan struct{}) error {
 		klog.Errorf("failed to traverse plugin socket path %q, err: %v", w.path, err)
 	}
 
-	// Traverse deprecated plugin dir, if specified.
-	if len(w.deprecatedPath) != 0 {
-		if err := w.traversePluginDir(w.deprecatedPath); err != nil {
```

Comment:
fyi reviewers - here is the main behavior change. there was a lot of plumbing and test changes